<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Light3DDock</class>
 <widget class="QWidget" name="Light3DDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>782</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLight">
      <attribute name="title">
       <string>Light</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="17" column="0">
        <widget class="QLabel" name="lLightDirection">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lLightIntensity">
         <property name="text">
          <string>Intensity</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="20" column="2">
        <widget class="QSpinBox" name="sbLightConeAngle">
         <property name="suffix">
          <string>°</string>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>368</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="19" column="2">
        <widget class="QSpinBox" name="sbLightAzimuth">
         <property name="suffix">
          <string>°</string>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="lLightConeAngle">
         <property name="text">
          <string>Cone angle</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="lLightAzimuth">
         <property name="text">
          <string>Azimuth</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="sbYFocalPoint">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QDoubleSpinBox" name="sbXPosition">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QDoubleSpinBox" name="sbLightIntensity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="18" column="2">
        <widget class="QSpinBox" name="sbLightElevation">
         <property name="suffix">
          <string>°</string>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="lLightElevation">
         <property name="text">
          <string>Elevation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Focal Point</string>
         </property>
        </widget>
       </item>
       <item row="15" column="2">
        <widget class="KColorButton" name="kcbLightSpecularColor"/>
       </item>
       <item row="16" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QDoubleSpinBox" name="sbZFocalPoint">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="lLightSpecularColor">
         <property name="text">
          <string>Specular </string>
         </property>
        </widget>
       </item>
       <item row="14" column="2">
        <widget class="KColorButton" name="kcbLightDiffuseColor"/>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="lLightDiffuseColor">
         <property name="text">
          <string>Diffuse</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="sbXFocalPoint">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="KColorButton" name="kcbLightAmbientColor"/>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="lLightAmbientColor">
         <property name="text">
          <string>Ambient</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="lLightColor">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QDoubleSpinBox" name="sbYPosition">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QDoubleSpinBox" name="sbZPosition">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>sbXFocalPoint</tabstop>
  <tabstop>sbYFocalPoint</tabstop>
  <tabstop>sbZFocalPoint</tabstop>
  <tabstop>sbXPosition</tabstop>
  <tabstop>sbYPosition</tabstop>
  <tabstop>sbZPosition</tabstop>
  <tabstop>sbLightIntensity</tabstop>
  <tabstop>kcbLightAmbientColor</tabstop>
  <tabstop>kcbLightDiffuseColor</tabstop>
  <tabstop>kcbLightSpecularColor</tabstop>
  <tabstop>sbLightElevation</tabstop>
  <tabstop>sbLightAzimuth</tabstop>
  <tabstop>sbLightConeAngle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
